src/frontends/tensorflow/src/utils.hpp

```diff
@@ -83,120 +81,13 @@ void get_const_input(const NodeContext& node, int64_t input_index, std::vector<T
-        return;
-        //}
-    }
-    const auto tensor_content_size = tensor_proto.tensor_content().size();
-    if (tensor_content_size % sizeof(VecT)) {
-        std::cerr << "[ ERROR ] tensor_content_size (" << tensor_content_size << ") is not a multiple of "
-                  << sizeof(VecT);
-    }
-
-    // If tensor_content_size is zero, we'll have to take the values from
-    // int_val, float_val, etc.
-    if (tensor_content_size == 0) {
-        int64_t n_elements = 1;
-        for (auto i = 0; i < shape.dim_size(); i++) {
-            TENSORFLOW_OP_VALIDATION(node,
-                                     shape.dim(i).size() >= 0,
-                                     "Const node has empty tensor and an unknown dimension size");
-            n_elements *= shape.dim(i).size();
-        }
-        values->resize(n_elements);
-
-        auto val_lastsaved = (T)0;  // cast
-        for (auto i = 0; i < n_elements; i++) {
-            int64_t val_size = 0;
-            auto val_i = (T)0;  // cast
-            switch (dt) {
-            // TODO: there are more element types to support
-            // here
-            case ::tensorflow::DT_INT32:
-                val_size = tensor_proto.int_val_size();
-                if (val_size > 0)
-                    val_i = tensor_proto.int_val()[i];
-                break;
-            case ::tensorflow::DT_INT64:
-                val_size = tensor_proto.int64_val_size();
-                if (val_size > 0)
-                    val_i = tensor_proto.int64_val()[i];
-                break;
-            case ::tensorflow::DT_FLOAT:
-                val_size = tensor_proto.float_val_size();
-                if (val_size > 0)
-                    val_i = tensor_proto.float_val()[i];
-                break;
-            case ::tensorflow::DT_BOOL:
-                val_size = tensor_proto.bool_val_size();
-                if (val_size > 0)
-                    val_i = tensor_proto.bool_val()[i];
-                break;
-            case ::tensorflow::DT_DOUBLE:
-                val_size = tensor_proto.double_val_size();
-                if (val_size > 0)
-                    val_i = tensor_proto.double_val()[i];
-                break;
-            default:
-                OPENVINO_DEBUG << "Const node has empty tensor_proto and we don't know how to "
-                                  "handle this element type";
-                FRONT_END_THROW("Encountered unknown element type " + DataType_Name(dt) + " on an empty tensor_proto");
-            }
-            if (val_size == 0) {
-                (*values)[i] = static_cast<T>(0);
-            } else if (i < val_size) {
-                (*values)[i] = val_i;
-                val_lastsaved = val_i;
-            } else {
-                (*values)[i] = val_lastsaved;
-            }
-        }
-    } else {
-        return;
-    }
-}
-
 template <typename T, typename VecT = T>
```

Comment:
please clean-up includes above. For example, `graph_iterator_proto.hpp` is not needed and it can contain protobuf specifics.